src/styles/Banner.css

```diff
@@ -32,16 +32,15 @@
     height: 100vh;
     height: calc(var(--vh, 1vh) * 100);
   }
-
-  .banner-content {
-    padding-bottom: 50px;
+  .banner-Buttons{
+    transform: translateY(-50px);
   }
 }
 
 .banner-header1 {
   color: black;
   text-transform: uppercase;
-  font-size: 80px;
+  font-size: 70px;
```

Comment:
I asked for `I want the first box to go up by some pixels with increased font size`
- why is the font-size reduced here  ?